### PR TITLE
Bl 324 as a developer i need a word bank to generate data

### PR DESCRIPTION
**Pull request by yestrella14**
# Description

Create a word bank for synthetic data generation

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by caitlinkermen — DISMISSED** [review]

**Comment by kathrynhodges** [comment]
line 4 here
structured_experiments/2022.05.02.synthetic_generation_word_bank/README.md

Could read something like 
"This dictionary contains only:"  on a separate line before the start of the description of words included.

The way it read to me felt like an unfinished sentance

File: structured_experiments/2022.02.17.synthetic_generation/docker-compose.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: "3"
 services:
   train_test_experiment:
     image: test_experiment
-    container_name: test_experiment
+    container_name: test_experiment2
```
Comment (jinjahninjah) [inline structured_experiments/2022.02.17.synthetic_generation/docker-compose.yml]: Moving forward, try to make the `container_name` more informational like `synthetic_experiment` just to help keep organized

No need to change, just something to ponder moving forward 👍

**Review by jinjahninjah — DISMISSED** [review]
Looks great! I'm surprised the synthetic data wasn't helpful as hypothesized, but that's just as valuable! Hurray for science!

File: structured_experiments/2022.02.17.synthetic_generation/docker-compose.yml (hunk 1 above)
Comment (yestrella14) [reply structured_experiments/2022.02.17.synthetic_generation/docker-compose.yml]: Thank you

**Review by jinjahninjah — APPROVED** [review]

**Review by nlittlepoole — APPROVED** [review]
Great Work!

🚢

**Comment by yestrella14** [comment]
Link to Loom video:

https://www.loom.com/share/b0e3f2e8d04b4296b870662e378cccfb

**Pull request merged**